cibuildwheel/resources/pinned_docker_images.cfg

```
@@ -1,27 +1,27 @@
 [x86_64]
 manylinux1 = quay.io/pypa/manylinux1_x86_64:2021-04-05-14df3a6
 manylinux2010 = quay.io/pypa/manylinux2010_x86_64:2021-02-06-3d322a5
-manylinux2014 = quay.io/pypa/manylinux2014_x86_64:2021-04-05-b4fd19d
-manylinux_2_24 = quay.io/pypa/manylinux_2_24_x86_64:2021-04-05-b4fd19d
+manylinux2014 = quay.io/pypa/manylinux2014_x86_64:2021-05-01-28d233a
+manylinux_2_24 = quay.io/pypa/manylinux_2_24_x86_64:2021-05-01-28d233a
 
 [i686]
 manylinux1 = quay.io/pypa/manylinux1_i686:2021-04-05-14df3a6
```

```suggestion
manylinux1 = quay.io/pypa/manylinux1_i686:2021-05-05-b64d921
```
Commit just before `auditwheel==4.0.0`
Should we ?